javascript/builtins/BigInt.json

```diff
@@ -0,0 +1,67 @@
+{
+  "javascript": {
+    "builtins": {
+      "BigInt": {
+        "__compat": {
+          "mdn_url": " https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/BigInt",

```

Comment:
The leading space before `http` is causing the linter to fail. Removing the space should fix it